### PR TITLE
combine mutiple rcv3 steps into one

### PR DESCRIPTION
**Pull request by manishtomar**
implements #1878.

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-e8a105d4d3/docs/autoscale/v1/developer-guide/)

**Comment by manishtomar** [comment]
jenkins please run integration tests

**Comment by manishtomar** [comment]
@glyph review please.

File: otter/convergence/transforming.py [hunk 1]
```diff
@@ -66,6 +68,28 @@ def optimize_steps(clb_steps):
 _register_bulk_clb_optimizer(RemoveNodesFromCLB, 'node_ids')
 
 
+def _register_bulk_rcv3_optimizer(step_class):
+    """
+    Merge together multiple RCv3 bulk steps.
+
+    :param step_class: One of :obj:`BulkAddToRCv3` or :obj:`BulkRemoveFromRCv3`
```
Comment (glyph) [inline otter/convergence/transforming.py]: A specific type signature would be helpful here. `:param type step_class:` would highlight the fact that you literally want a `type` object.

**Comment by glyph** [comment]
This looks super straightforward, the tests make it very clear what's happening; 👍

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-8e95a82e50/docs/autoscale/v1/developer-guide/)

**Pull request merged**